src/libcore/char.rs

```diff
@@ -306,6 +306,7 @@ impl CharExt for char {
     fn escape_default(self) -> EscapeDefault {
         let init_state = match self {
             '\t' => EscapeDefaultState::Backslash('t'),
+            '\x1B' => EscapeDefaultState::Backslash('e'),
```

Comment:
this literal could be changed to `'\e` after a snapshot, correct? might be worth noting